### PR TITLE
Subscription

### PR DESCRIPTION
**Pull request by bananoid**

### CODE REVIEW

**Comment by justuswilhelm** [comment]
Does the deploy preview work for you? I'm getting this error in my js console
```
Firefox can’t establish a connection to the server at wss://api.projectifyapp.com/graphql-ws. vendor-f436a9d8.js:87:21871
The connection to wss://api.projectifyapp.com/graphql-ws was interrupted while the page was loading.
```

File: src/messages/en.json [hunk 1]
```diff
@@ -1,3 +1,58 @@
...
+    "404": "404",
+    "home": "Home",
+    "signout": "Signout",
+    "signup": "Signup",
+    "signin": "Signin",
+    "dashboard": "Dashoard",
+    "welcome-home": "Welcome Home",
+    "sign-up": "Sign up",
+    "page.signup.msg": "Let’s set up your account, Already have one?",
+    "sing-in-here": "Sign in here.",
```
Comment (justuswilhelm) [inline src/messages/en.json:11]:
```suggestion
    "sign-in-here": "Sign in here.",
```

File: src/messages/en.json [hunk 2]
```diff
@@ -1,3 +1,58 @@
...
+    "password": "Password",
+    "please-enter-a-password": "Please enter a password",
+    "user-already-exist": "User already exist.",
+    "signup-privacy": "I accept the Projectify Terms of Service. For more information about Projectify's use and protection of your data, please see our Privacy Policy.",
+    "email-sent": "Email sent",
+    "i-sent-an-email-to": "I sent an email to",
+    "please-proceed-from-the-url-described-in-the-message": "Please proceed from the url described in the message.",
+    "top-page": "Top page",
+    "password-reset": "Password reset",
+    "page-user-reset-msg": "Please enter a your email to request a password reset.",
```
Comment (justuswilhelm) [inline src/messages/en.json:23]:
```suggestion
    "page-user-reset-msg": "Please enter your email to request a password reset.",
```

File: src/messages/en.json [hunk 3]
```diff
@@ -1,3 +1,58 @@
...
+    "user-not-found": "User not found.",
+    "send": "Send",
+    "password-reset-email-sent": "Password reset email sent",
+    "please-enter-your-email-to-request-a-password-reset": "Please enter your email to request a password reset.",
```
Comment (justuswilhelm) [inline src/messages/en.json:27]: Isn't this the same as the message above in line 23?

File: src/messages/en.json [hunk 4]
```diff
@@ -1,3 +1,58 @@
...
+    "password-can-not-be-empty": "Password can not be empty.",
+    "something-went-wrong": "Something went wrong.",
+    "password-reset-complete": "Password reset complete",
+    "user-password-reset-completed-msg": "Your password has been changed, and you have been logged into your account. Please continue to enjoy Projectify.",
```
Comment (justuswilhelm) [inline src/messages/en.json:31]: I thought, when the password changes you have to log in one more time

File: src/messages/en.json [hunk 5]
```diff
@@ -1,3 +1,58 @@
...
+    "lets-set-up-your-account-dont-have-one-yet": "Let's set up your account, Don't have one yet?",
+    "sign-up-here": "Sign up here.",
+    "wrong-email-or-password": "Wrong email or password.",
+    "forget-password-password-reset": "Forget password? Password reset",
+    "here": "here.",
+    "task-name": "Task Name",
+    "save": "Save",
+    "task": "Task",
+    "discussion": "Discussion",
+    "overview": "Overview",
```
Comment (justuswilhelm) [inline src/messages/en.json:46]:
```suggestion
    "description": "Description",
```

File: src/messages/en.json [hunk 6]
```diff
@@ -1,3 +1,58 @@
...
+    "please-enter-an-overview": "Please enter an overview",
```
Comment (justuswilhelm) [inline src/messages/en.json:47]:
```suggestion
    "please-enter-a-description": "Please enter a description",
```

File: src/routes/signup.svelte [hunk 7]
```diff
@@ -51,25 +52,27 @@
...
-                        <h1 class="card-title">Sign up</h1>
+                        <h1 class="card-title">
+                            {$_("sign-up")}
+                        </h1>
                         <div>
-                            Let’s set up your account, Already have one?<wbr />
+                            {$_("page.signup.msg")}<wbr />
                             <a class="link link-primary" href="/signin">
-                                Sign in here.
+                                {$_("sing-in-here")}
```
Comment (justuswilhelm) [inline src/routes/signup.svelte:61]:
```suggestion
                                {$_("sign-in-here")}
```

**Review by justuswilhelm — CHANGES_REQUESTED** [review]
Some small translation nitpicks

File: src/messages/en.json (hunk 3 above)
Comment (bananoid) [reply src/messages/en.json:26]: yes I merged

File: src/messages/en.json (hunk 4 above)
Comment (bananoid) [reply src/messages/en.json:30]: Yes, I just copy paste from the mockup. Witch phrase can I use?

Comment (justuswilhelm) [reply src/messages/en.json:30]:
```suggestion
    "user-password-reset-completed-msg": "Your password has been changed. Please log in to continue using Projectify.",
```

**Review by justuswilhelm — APPROVED** [review]
LGTM thanks!

**Pull request merged**